recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston

```diff
@@ -287,7 +287,7 @@ LICENSE_FLAGS_WHITELIST = "commercial"
 DISTRO_FEATURES_append = " mm-test"
 
 # for weston v4l2 renderer
-#DISTRO_FEATURES_append = " v4l2-renderer"
+DISTRO_FEATURES_append = " v4l2-renderer"
```

Comment:
As we agreed this shouldn't be enabled for all machines, but for H3 ES3.0 based board only.

Reply:
I will fix.